src/modules/data/entities/aggregate/global/global.character.aggregate.entity.ts

```diff
@@ -100,4 +100,18 @@ export default class GlobalCharacterAggregateEntity {
         default: Ps2AlertsEventType.LIVE_METAGAME,
     })
     ps2AlertsEventType: Ps2AlertsEventType;
+
+    @Exclude()
```

Comment:
Prevents this being persisted to the entity but allows us to add it to objects in results